src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -778,10 +837,6 @@ class ResourceService(
       changeEvents = createAccessChangeEvents(policyId.resource, originalPolicies, updatedPolicies)
       _ <- AuditLogger.logAuditEventIO(samRequestContext, changeEvents.toSeq: _*)
 
-      _ <- directoryDAO.updateGroupUpdatedDateAndVersionWithSession(
-        FullyQualifiedPolicyId(policyId.resource, policyId.accessPolicyName),
-        samRequestContext
-      )
```

Comment:
I think this was here as a failsafe but was double doing work. I made sure it is correctly called in the DAO. Maybe a follow on would be to figure out a trigger even though I despise triggers.